torch/utils/cpp_extension.py

```diff
@@ -741,11 +746,20 @@ def load_inline(name,
 
     cpp_sources.insert(0, '#include <torch/extension.h>')
 
+    def add_safe_version(function_name):
+        cpp_sources.append('template <typename ...Args>')
+        cpp_sources.append('auto _safe_{0}(Args ...args) -> decltype({0}(args...)) {{'.format(function_name))
+        cpp_sources.append('HANDLE_TH_ERRORS')
+        cpp_sources.append('return {0}(args...);'.format(function_name))
```

Comment:
I think you should use perfect forwarding here. So `Args&& ...` and then `std::forward<Args>(args...)` (or something like that)